updater/Gemfile.lock

```diff
@@ -201,7 +201,7 @@ GEM
       rake (~> 13.0)
     mime-types (3.4.1)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2022.0105)
+    mime-types-data (3.2023.0218.1)
```

Comment:
Why did this change? 🤔 

None of the changes landed in this release bump any deps AFAICT.

Unless it's the issue that the updater script doesn't actually pickup all the PRs in the changelog... part of why switching to GitHub releases... 

(and 🙇‍♂️ for cutting this release to get it out, even though a little more painful right now)

Reply:
I'm going to revert this change as well, I suspect this is due to me running the update on my mac and not in a codespace.

Reply:
I did merge a number of dependabot PRs, but I didn't think any to do with mime-types.